sdk/api/sdk.api

```diff
@@ -0,0 +1,265 @@
+	public fun identify (Ljava/lang/String;Ljava/util/Map;)V
+	public static final fun instance ()Lio/customer/sdk/CustomerIO;
+	public fun registerDeviceToken (Ljava/lang/String;)V
+	public fun screen (Landroid/app/Activity;)V
+	public fun screen (Landroid/app/Activity;Ljava/util/Map;)V
+	public fun screen (Ljava/lang/String;)V
+	public fun screen (Ljava/lang/String;Ljava/util/Map;)V
+	public fun setDeviceAttributes (Ljava/util/Map;)V
+	public fun setProfileAttributes (Ljava/util/Map;)V
+	public fun track (Ljava/lang/String;)V
+	public fun track (Ljava/lang/String;Ljava/util/Map;)V
+	public fun trackMetric (Ljava/lang/String;Lio/customer/sdk/data/request/MetricEvent;Ljava/lang/String;)V
+}
+
+public final class io/customer/sdk/CustomerIO$Builder {
+	public fun <init> (Ljava/lang/String;Ljava/lang/String;Landroid/app/Application;)V
+	public fun <init> (Ljava/lang/String;Ljava/lang/String;Lio/customer/sdk/data/model/Region;Landroid/app/Application;)V
+	public synthetic fun <init> (Ljava/lang/String;Ljava/lang/String;Lio/customer/sdk/data/model/Region;Landroid/app/Application;ILkotlin/jvm/internal/DefaultConstructorMarker;)V
+	public final fun addCustomerIOModule (Lio/customer/sdk/CustomerIOModule;)Lio/customer/sdk/CustomerIO$Builder;
+	public final fun autoTrackDeviceAttributes (Z)Lio/customer/sdk/CustomerIO$Builder;
+	public final fun autoTrackScreenViews (Z)Lio/customer/sdk/CustomerIO$Builder;
+	public final fun build ()Lio/customer/sdk/CustomerIO;
+	public final fun setCustomerIOUrlHandler (Lio/customer/sdk/data/communication/CustomerIOUrlHandler;)Lio/customer/sdk/CustomerIO$Builder;
+	public final fun setLogLevel (Lio/customer/sdk/util/CioLogLevel;)Lio/customer/sdk/CustomerIO$Builder;
+	public final fun setRegion (Lio/customer/sdk/data/model/Region;)Lio/customer/sdk/CustomerIO$Builder;
+	public final fun setRequestTimeout (J)Lio/customer/sdk/CustomerIO$Builder;
+	public final fun setTrackingApiURL (Ljava/lang/String;)Lio/customer/sdk/CustomerIO$Builder;
+}
+
+public final class io/customer/sdk/CustomerIO$Companion {
+	public final fun instance ()Lio/customer/sdk/CustomerIO;
+}
+
+public final class io/customer/sdk/CustomerIOConfig {
+	public fun <init> (Ljava/lang/String;Ljava/lang/String;Lio/customer/sdk/data/model/Region;JLio/customer/sdk/data/communication/CustomerIOUrlHandler;ZZIDDLio/customer/sdk/util/CioLogLevel;Ljava/lang/String;)V
+	public final fun component1 ()Ljava/lang/String;
+	public final fun component10 ()D
+	public final fun component11 ()Lio/customer/sdk/util/CioLogLevel;
+	public final fun component12 ()Ljava/lang/String;
+	public final fun component2 ()Ljava/lang/String;
+	public final fun component3 ()Lio/customer/sdk/data/model/Region;
+	public final fun component4 ()J
+	public final fun component5 ()Lio/customer/sdk/data/communication/CustomerIOUrlHandler;
+	public final fun component6 ()Z
+	public final fun component7 ()Z
+	public final fun component8 ()I
+	public final fun component9 ()D
```

Comment:
???

I expected this to setters for each property (setApiKey for example). Is this what Java customers see? Is this a bug with the binary validator plugin?